.changeset/nasty-candles-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
+'@sap-ux/preview-middleware': patch
```

Comment:
```suggestion
```
No longer needed

Reply:
I think we need both since `@sap-ux-private/preview-middleware-client` is not released but just packaged. If we remove the patch for the other one then we wouldn't get a meaningful changelog entry for the released module.

Reply:
ok